src/Elasticity/Elasticity.jl

```diff
@@ -72,6 +79,15 @@ function param_info(s::ConstantElasticity) # info about the struct
     return MaterialParamsInfo(; Equation=L"Constant elasticity")
 end
 
+function isvolumetric(a::ConstantElasticity)
+    @unpack_val ν = a
+    if ν == 0.5
```

Comment:
just for the sake of having less lines of code, this can be just `return ν == 0.5 ? false : true`